src/velero/values/values.yaml

```diff
@@ -0,0 +1,56 @@
+kubectl:
+  # Container Level Security Context for the 'kubectl' container of the crd jobs. Optional.
+  # See: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  containerSecurityContext:
+    capabilities:
+      drop:
+        - ALL
+  # Annotations to set for the upgrade/cleanup job. Optional.
+  annotations:
+    sidecar.istio.io/inject: 'false'
+
+credentials:
+  useSecret: true
+  name: "velero-bucket-credentials"
+  secretContents:
+    cloud: |
+      [default]
+      aws_access_key_id=###ZARF_VAR_VELERO_BUCKET_KEY###
+      aws_secret_access_key=###ZARF_VAR_VELERO_BUCKET_KEY_SECRET###
+configuration:
+  backupStorageLocation:
+    - name: default
+      provider: aws
+      bucket: "###ZARF_VAR_VELERO_BUCKET###"
+      config:
+        region: "###ZARF_VAR_VELERO_BUCKET_REGION###"
+        s3ForcePathStyle: true
+        s3Url: "###ZARF_VAR_VELERO_BUCKET_PROVIDER_URL###"
+      credential:
+        name: "velero-bucket-credentials"
+        key: "cloud"
+  # volumeSnapshotLocation:
+  #   - name: default
+  #     provider: aws
+  #     config:
+  #       region: "###ZARF_VAR_VELERO_BUCKET_REGION###"
+  #       s3ForcePathStyle: true
+  #       s3Url: "###ZARF_VAR_VELERO_BUCKET_PROVIDER_URL###"
+  #     credential:
+  #       name: "velero-bucket-credentials"
+  #       key: "cloud"
+snapshotsEnabled: false
+schedules:
+  udsbackup:
+    disabled: false
+    schedule: "0 3 * * *"
+    useOwnerReferencesInBackup: false
+    template:
+      csiSnapshotTimeout: 0s
+      includeClusterResources: true
+      snapshotVolumes: false
+      excludedNamespaces:
+        - kube-system
+        - flux
+        - velero
```

Comment:
Cross referencing the [config we did for DUBBD](https://github.com/defenseunicorns/uds-package-dubbd/blob/main/aws/values/aws-velero.yaml#L26-L31) - do you see any plusses/minuses to exclude vs include here? If we stick with exclude we can drop `flux`.

For what its worth I think excluded makes more sense so we backup things on top of uds-core as well?

Reply:
Depends which [dubbd config you crossreference](https://github.com/defenseunicorns/uds-package-dubbd/blob/main/rke2/values/rke2-velero-s3.yaml) 🙈 

That said.. I think exclude makes the most sense as a DEFAULT config. As you said, I think the intention would be to capture a backup of everything on top of uds-core.

In practice, I fully expect downstream users to override this config to meet environment specific requirements so I would almost treat this one as more of a simple example..

Reply:
:+1: seems legit, and yeah expect that others will override to customize what, where, when they backup things.